content-resources/src/main/java/fi/nls/oskari/db/DBHandler.java

```diff
@@ -95,66 +93,93 @@ public static void createContentIfNotCreated() {
             createContentIfNotCreated(getDataSource());
         }
         catch (Exception ex) {
-            log.error(ex, "Couldn't get connection to db!");
+            getLog().error(ex, "Couldn't get connection to db!");
```

Comment:
Exception handling in this class is all over the place. However those lines aren't under review, perhaps we should leave them be.

Reply:
There's also few instances of System.out.println calls in this class :worried:

Reply:
Yep, it can be used as a standalone jar-file or as part of a webapp to setup the database content which makes error handling and logging a bit trickier and changes would require more testing. Perhaps some day this can have a better overhaul, but for now the main issue is to fix the initial database creation as it's not working at all without changes.